### ISSUE TITLE
Doxygen documentation missing?

### ISSUE THREAD

**Issue by cpdupuis** [adafruit/Adafruit_VL53L0X#44]
Documentation page at http://adafruit.github.io/Adafruit_VL53L0X/html/index.html appears to be missing the documentation.

**Comment by ladyada** [comment]
yeah we have not completed the doxygen documentation for this library so it is not generated :)
